site/index.css

```
@@ -67,6 +67,17 @@ body {
     font-size: 1em;
   }
 }
+@media screen and (max-width: 1420px) {
```

Why limit the change to screen sizes under the width of 1420px? Did you mean to make this greater than 1420px?

For the example given in #13 this media query would not apply. We could use a `min-width` media query here instead?

```suggestion
@media screen and (min-width: 1420px) {
```

---

Yes @aredshift, I made a mistake, I should've used min-width instead of max-width.

---

No worries! Would you mind pushing another commit to fix this?

---

Right away!